model.py

```diff
@@ -1224,6 +1232,8 @@ class Hyperlink(Base):
         UniqueConstraint('library_id', 'rel'),
     )
 
+    TIME_FORMAT = "%Y-%m-%d %H:%M:%S UTC"
```

Comment:
I don't think this is a very good choice for the time format. I actually might just say the link expires in X days instead of specifying the exact time.

Reply:
I took this out. The link expires one day from the time it was created, and since the email might only be checked once a day it's not really useful to talk in terms of days. In an earlier version of this branch the link could be sent out multiple times and this was more important.